userpanel/modules/helpdesk/functions.php

```diff
@@ -194,7 +194,6 @@ function module_main()
                 'queue' => $ticket['queue'],
                 'subject' => $ticket['subject'],
                 'customerid' => $SESSION->id,
-                'requestor' => $LMS->GetCustomerName($SESSION->id),
```

Comment:
A co powiesz na to:
https://github.com/chilek/lms-plus/blob/d4dd4ac4c8c22503a6d31b2e44b635a0d6d4cd47/lib/LMSManagers/LMSHelpdeskManager.php#L1089
?

Reply:
Przyznam się, że zupełnie nie spodziewałem się wymagalności tego parametru w tej metodzie. Dorzucę do metody fragment kodu od razu w tych trzech liniach żeby nie trzeba było przekazywać pustych parametrów.

Reply:
a2038eba31d3706d53096e87632fb4826aaddf6a - proszę